pkgs/os-specific/linux/xone/default.nix

```
@@ -11,6 +11,15 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-srAEw1ai5KT0rmVUL3Dut9R2mNb00AAZVCcINikh2sM=";
   };
 
+  patches = [
+    # Fix build on kernel 6.11
+    (fetchpatch {
+      name = "kernel-6.11.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/medusalix/xone/pull/48.patch";
```

please use a reproducible url

```suggestion
      url = "https://github.com/medusalix/xone/commit/28df566c38e0ee500fd5f74643fc35f21a4ff696.patch";
```

---

Fixed.